decidim-assemblies/config/locales/en.yml

```
@@ -247,7 +247,11 @@ en:
             valuator: Valuator
       titles:
         assemblies: Assemblies
+        assemblies_setting: Assemblies settings
         assemblies_types: Assemblies types
+        assemblies_types_edit: Assemblies types edit
```

Let's use the singular for this value: 
```suggestion
        assemblies_types_edit: Edit assembly type
```